lib/profiles/dosomething/drupal-org.make

```diff
@@ -275,6 +275,10 @@ libraries[predis][download][url] = "https://github.com/nrk/predis.git"
 ; Pin to 0.8 release, which still has the lib directory.
 libraries[predis][download][revision] = "4123fcd85d61354c6c9900db76c9597dbd129bf6"
 
+; Stathat
+libraries[stathat][download][type] = "get"
```

Comment:
Ok then. This is strange though, they didn't include this type to `drush topic docs-make`.